lib/diagnostic.js

```diff
@@ -29,29 +33,30 @@ const sum = function (/* previous, current */) {
 // in the array returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.some`
-const any = function (/* array, callback */) {
-
+const any = function isBigEnough(element, index, array) {
+    return element >= 10;
 };
+[12, 54, 18, 130, 44].every(isBigEnough);
```

Comment:
It seems that you misunderstood the question. The question is asking you to write a function that returns true if the callback returns true for any value that is passed into it. One hint: go back to the lesson and look at the `.some` method that we wrote together.

Please take a look at this solution:
```
const any = function (array, callback) {
  return !array.every((e, i, a) => !callback(e, i, a));
};
```